cppcache/src/statistics/StatisticsTypeImpl.cpp

```diff
@@ -137,7 +132,7 @@ int32_t StatisticsTypeImpl::getDoubleStatCount() const {
   return doubleStatCount;
 }
 
-int32_t StatisticsTypeImpl::getDescriptorsCount() const { return statsLength; }
+int32_t StatisticsTypeImpl::getDescriptorsCount() const { return stats.size(); }
```

Comment:
Getting this build gripe on Windows, please fix with type cast (or change return type to size_t?)
```
Z:\nativeclient\cppcache\src\statistics\StatisticsTypeImpl.cpp(135): warning C4267: 'return': conversion from 'size_t' to 'int32_t', possible loss of data [Z:\build\cppcache\static\apache-geode-static.vcxproj]
```

Reply:
ok. Thanks. I will type cast the returned value.

Reply:
You can safely change the return type to `size_t` since this is an internal class.

Reply:
> You can safely change the return type to `size_t` since this is an internal class.

ok